src/utills.js

```diff
@@ -0,0 +1,19 @@
+const grab = (e) => document.getElementById(e);
+
+function openModal(modal, callback) {
+  const element = document.getElementsByClassName('js-modal')[modal];
+  const event = new CustomEvent('openModal');
+  element.dispatchEvent(event);
+
+  element.addEventListener('modalIsClose', () => { callback(); });
+}
+
+const dspCommentsLength = (comments) => comments.length;
+
+function create(e) {
+  return document.createElement(e);
+}
```

Comment:
- [ ] Here as well, Use `arrow function` instead of using `function` keyword. We have supposed to use `ES6` syntax which improves the readability and it is best for callbacks as well.